pages/1.12/administering-clusters/package-registry/using/index.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: layout.pug
+navigationTitle: Using Package Registry
+title: Using Package Registry
+menuWeight: 5
+excerpt: How to Use DC/OS Package Registry
+beta: true
+enterprise: true
+---
+
+### Installing DC/OS Package Registry CLI
+
+Installing the DC/OS Package Registry also installs the CLI for it (`dcos registry`). The CLI can be installed again with:
+
+```bash
+dcos package install package-registry --cli
+```
+
+### Downloading Packages
+
+For a list of DC/OS Packages supported by the DC/OS Package Registry see here. To download a version of Jenkins:
```

Comment:
`here` is not a valid hyperlink. Should refer to https://downloads.mesosphere.com/universe/packages/packages.html

Reply:
got it